File: Ugly Number/file.py
def ugly(n):
    if(n%2==0):
        return ugly(int(n/2))
    elif(n%3==0):
        return ugly(int(n/3))
    elif(n%5==0):
        return ugly(int(n/5))
    elif(n==1):
        return 1
    else:
        return 0
    

num=int(input())
print("No it's not an Ugly Number") if(ugly(num)==0) else print("Yes it's an Ugly Number")
    

File: Ugly Number/file.c
#include <stdio.h>
int ugly(int n){
    int flag=1;
    while(n>1){
        if(n%2==0){
            n/=2;
        }
        else if(n%3==0){
            n/=3;
        }
        else if(n%5==0){
            n/=5;
        }
        else{
            flag=0;
            break;
        }
    }
    return flag;
}
int main()
{
    int num;
    scanf("%d",&num);
    int x=ugly(num);
    x==1 ? printf("Yes it's an Ugly Number"):printf("No it's not an Ugly Number");
    
}
